_posts/2020-04-13-new-home-totalopenstation-project.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "A new home for the Total Open Station project"
+date: 2020-04-13 22:10 CEST
+author: steko
+---
+
+As announced in the previous blog post, the [Total Open Station project has a new home](https://github.com/totalopenstation/totalopenstation)!
+
+More specifically, we're still on GitHub but I have moved the main repository from my personal GitHub account to the "totalopenstation" organization, with myself and @psolyca as owners.
+
+This may seem like a simple step but, in fact, it is the most important improvement that Total Open Station has ever got. 
+
+Before this change, we put a lot of effort to define how the new project will work, with a detailed [CONTRIBUTING](https://github.com/totalopenstation/totalopenstation/blob/master/CONTRIBUTING.md) document (partly inspired by the great example of the [Gitea project](https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md)) and a [Code of Conduct](https://github.com/totalopenstation/totalopenstation/blob/master/CODE_OF_CONDUCT.md) (based on the Contributor Covenant). To ensure code quality, all changes must be approved through a review process before being merged into the master branch. This is enforced for all contributors, myself included! There is a process in place to expand the team of owners with new contributors.
```

Comment:
[...] project will work. We have added a detailed [...]

There is a process in place to expand the team with new contributors on any part of the application, code, provide samples, design...